### ISSUE TITLE
Logstash 6.4 fails to start due to issues with Kafka plugin

### ISSUE THREAD

**Issue by LeanVel** [rocknsm/rock#316]
After installing RockNSM 2.1 (online) and updating the system, Logstash throw the following error message:
```
[ERROR][logstash.outputs.kafka ] Unable to create Kafka producer from given configuration {:kafka_error_message=>org.apache.kafka.common.config.ConfigException: Invalid value 32768 for configuration receive.buffer.bytes: Expected value to be a 32-bit integer, but it was a java.lang.Long, :cause=>nil}
```

It seems that this error is caused when the Elastic Stack gets updated to 6.4 (during the yum update), as described here https://discuss.elastic.co/t/kafka-output-plugin-java-lang-long-error/145398.

I tried the solution proposed in the link (downgrading the logstash plugin for kafka) and it did work for me. 

```bash
bin/logstash-plugin uninstall logstash-output-kafka
bin/logstash-plugin install --version 7.1.1 logstash-output-kafka
```
I guess this compatibility issue will be solved with the release of the next version of the plugin, however, in the meanwhile this workaround needs to be applied.

**Comment by dcode** [comment]
@LeanVel thanks a ton for sharing! We were actually working on a fix for this since they pushed out 6.4 the day after we cut the 2.1.0 ISO. We're working on a hotfix release asap.

@jeffgeiger

**Comment by jeffgeiger** [comment]
Confirmed it. Outstanding work @LeanVel.

**Comment by jeffgeiger** [comment]
Added to the community site:
https://community.rocknsm.io/t/rock-2-1-and-elastic-6-4/89/2?u=el_jefe

**Comment by botaka68** [comment]
Fixed by updating Kafka plugin to 7.1.3.

**Comment by neu5ron** [comment]
@dcode @jeffgeiger close me :)
